build.gradle

```diff
@@ -0,0 +1,18 @@
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
+subprojects {
+    configurations {
+        ktlint
```

Comment:
I've decided to use `ktlint` instead of `detekt` as we agreed in FluxC or Tracks that it doesn't need any additional config and I wanted this initial setup to be a minimalistic one. Also, in such a small project, I think `ktlint` will do just fine.